### PR TITLE
🔧 Updated README to include CMS testing

### PR DESCRIPTION
**Pull request by AndrewJDick**
## Ticket

_Which GitHub issue does your code resolve?_

Closes: #391

## Proposed Changes

_What changes does your code introduce?_

- Outlines steps for testing CMS changes

## Types of changes

_What types of changes does your code introduce?_

- [ ] 🔥 Breaking change (fix or feature that would cause existing functionality to change)
- [ ] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🐛 Bug fix (non-breaking change which fixes an issue)
- [ ] 🎨 Improve general format/structure
- [ ] ⬆️ Upgrading dependencies
- [x] 🔧 Configuration

### CODE REVIEW

**Comment by peduarte** [comment]
Wasn't there a step about removing local storage stuff?

**Comment by AndrewJDick** [comment]
@peduarte the local storage was when we were trying to access the CMS from `localhost`.
Those steps in the PR are for accessing it from `deploy-preview-xxx`

File: README.md [hunk 1]
```diff
@@ -17,3 +17,16 @@ yarn develop
...
 yarn build
 ```
+
+## Testing
+
+### CMS Changes
+
+If any features require you to make changes to the CMS, you can test them on your feature branch by:
+
+1. Replacing `backend: branch: master` with your branch name in `static/admin/config.yml`.
```
Comment (peduarte) [inline README.md]: don't think we need `backend` here

File: README.md [hunk 2]
```diff
@@ -17,3 +17,16 @@ yarn develop
 ```
 yarn build
 ```
+
+## Testing
+
+### CMS Changes
```
Comment (peduarte) [inline README.md]: Can you add another level of title which explains the steps above are for editing the CMS in a feature build?

**Comment by peduarte** [comment]
The note below is from Netlify CMS docs. Could be helpful to have here, or link to it

https://www.netlifycms.org/docs/add-to-your-site/#accessing-the-cms

>Note: No matter where you access Netlify CMS — whether running locally, in a staging environment, or in your published site — it always fetches and commits files in your hosted repository (for example, on GitHub), on the branch you configured in your Netlify CMS config.yml file. This means that content fetched in the admin UI matches the content in the repository, which may be different from your locally running site. It also means that content saved using the admin UI saves directly to the hosted repository, even if you're running the UI locally or in staging.

**Comment by AndrewJDick** [comment]
@peduarte good to go 👍

**Pull request merged**